service.subtitles.legendasdivx/service.py

```diff
@@ -38,7 +38,7 @@
 _cwd        = xbmc.translatePath(_addon.getAddonInfo('path')).decode("utf-8")
 _profile    = xbmc.translatePath(_addon.getAddonInfo('profile')).decode("utf-8")
 _resource   = xbmc.translatePath(os.path.join(_cwd, 'resources', 'lib' )).decode("utf-8")
-_temp       = xbmc.translatePath(os.path.join(_profile, 'temp')).decode("utf-8")
+_temp       = xbmc.translatePath(os.path.join(_profile, 'temp'))
```

Comment:
hmm doesnt really make sense to me to .decode() some paths, and some others not? youre sure that youre not just hiding some other issues with this change?